data/json/npcs/BGSS_talk.json

```diff
@@ -0,0 +1,922 @@
+[{
+    "id": "TALK_LEADER",
+    "type": "talk_topic",
+    "responses": [
+	{ "text": "Tell me about how you survived (CONFUSED).", 
```

Comment:
Am I right that the stuff in parenthesis like (CONFUSED) is for your own notes and will get removed eventually?  Because I don't think the game engine knows not to remove (CONFUSED) from that line.

Also, consider using snippets here, so it would be
```
{
  "text": "<background_intro_question>", "topic": $TOPIC, ...
}
```
and then players will be able to ask for background stories in different ways and not know which response they'll get.

Reply:
Ah yes, those were meant to be gone in this version actually. They were just from me debugging, to quickly make sure the NPCs were selecting the right story.

Making the question a snippet is a cool idea. I didn't realize I could do that.